alleles/countSam.go

```diff
@@ -77,13 +77,13 @@ func sendPassedPositionsSam(answer chan<- *Allele, aln *sam.SamAln, samFilename
 }
 
 // countSamRead adds the bases in a single sam read to the currAlleles map
-func countSamRead(aln *sam.SamAln, currAlleles map[Coordinate]*AlleleCount, runningCount []*Coordinate, ref map[string][]dna.Base, minMapQ int64, progress int) (map[Coordinate]*AlleleCount, []*Coordinate) {
+func countSamRead(aln sam.Sam, currAlleles map[Coordinate]*AlleleCount, runningCount []*Coordinate, ref map[string][]dna.Base, minMapQ int64, progress int) (map[Coordinate]*AlleleCount, []*Coordinate) {
 
 	if aln.Cigar[0].Op == '*' {
 		return currAlleles, runningCount
 	}
 
-	if aln.MapQ < minMapQ {
+	if int64(aln.MapQ) < minMapQ {
```

Comment:
You are right of course. This is a patch job, but I don't think it is worth changing this value and dealing with the cascading effects down to the cmd calling this function (at least in this PR). Especially since the alleles package is likely to be entirely rewritten in the future.